### PR TITLE
[UR][DeviceSantizer] Enable Symoblizer for UR santizer layer

### PR DESCRIPTION
**Pull request by zhaomaosu**
Re-land PR #14513 

UR Part: https://github.com/oneapi-src/unified-runtime/pull/1920

### CODE REVIEW

File: sycl/cmake/modules/FetchUnifiedRuntime.cmake [hunk 1]
```diff
@@ -115,14 +116,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
...
 
-set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 3e762e00bcf13d158fb58e8e8c2eabcfc8934b4e
-  # Merge: c805a71a a2a053de
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Wed Jul 31 12:26:34 2024 +0100
-  #     Merge pull request #1884 from callumfare/callum/fix_printtrace
-  #     Enable PrintTrace when SYCL UR tracing is enabled
-  set(UNIFIED_RUNTIME_TAG 3e762e00bcf13d158fb58e8e8c2eabcfc8934b4e)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/zhaomaosu/unified-runtime.git")
```
Comment (pbalcer) [inline sycl/cmake/modules/FetchUnifiedRuntime.cmake:119]: patch was merged, please update the commit

Comment (zhaomaosu) [reply sycl/cmake/modules/FetchUnifiedRuntime.cmake]: Thanks. Updated.

Comment (zhaomaosu) [reply sycl/cmake/modules/FetchUnifiedRuntime.cmake]: Hi @pbalcer, looks like the UR tag is ahead of my fix, so I revert tag updating part. May I get your approval for this change? Thanks.

**Comment by zhaomaosu** [comment]
Kindly ping @pbalcer, @intel/unified-runtime-reviewers

**Review by pbalcer — APPROVED** [review]

**Comment by pbalcer** [comment]
@intel/llvm-gatekeepers please merge.

**Pull request merged**